src/sentry/tasks/commits.py

```diff
@@ -72,7 +72,9 @@ def fetch_commits(release_id, user_id, refs, prev_release_id=None, **kwargs):
     commit_list = []
 
     release = Release.objects.get(id=release_id)
-    user = User.objects.get(id=user_id)
+    # TODO: Need a better way to error handle no user_id. We need the SDK to be able to call this without user context
+    # to autoassociate commits to releases
+    user = User.objects.get(id=user_id) if user_id is not None else None
```

Comment:
Could use some suggestions on how to handle `user` here as we don't know what the acting user is at ingest.